views/js/controllers/configuration/back.js

```diff
@@ -26,97 +26,43 @@
-        let name = $(this).attr('data-name');
-        let action = $(this).attr('data-action');
-        let id_module = $('.src_parent_'+name).data('id_module');
-        ajaxActionModule(action, id_module, name);
-    });
-
-    $("#psthemecusto .js-wireframe div").hover(
-        function() {
-            $(this).find('.on-element').removeClass('displaynone');
-            $(this).find('.out-element').addClass('displaynone');
-        }, function() {
-            $(this).find('.on-element').addClass('displaynone');
-            $(this).find('.out-element').removeClass('displaynone');
-        }
-    );
+'use strict';
 
-});
-
-function resetActiveCategory()
-{
+let resetActiveCategory = function() {
     $('#psthemecusto .js-wireframe div').removeClass('active');
-    $('#psthemecusto .js-wireframe div .on-element').addClass('displaynone');
-    $('#psthemecusto .js-wireframe div .out-element').removeClass('displaynone');
+    $('#psthemecusto .js-wireframe div .on-element').addClass('hide');
+    $('#psthemecusto .js-wireframe div .out-element').removeClass('hide');
     $('#psthemecusto .js-module-name').removeClass('active');
     $('#psthemecusto .js-module-name').parent('.configuration-rectangle').removeClass('active');
     $('#psthemecusto .js-module-name').parent('.configuration-rectangle').find('.module-informations').slideUp();
     $('#psthemecusto .configuration-rectangle-caret .material-icons.up').hide();
     $('#psthemecusto .configuration-rectangle-caret .material-icons.down').show();
 }
 
-function setActiveCategory(elem)
-{
+let setActiveCategory = function(elem) {
     let module = elem.data('module_name');
     $('.js-img-'+module).addClass('active');
-    $('.js-img-'+module+' .on-element').removeClass('displaynone');
-    $('.js-img-'+module+' .out-element').addClass('displaynone');
+    $('.js-img-'+module+' .on-element').removeClass('hide');
+    $('.js-img-'+module+' .out-element').addClass('hide');
     $('.js-title-'+module).addClass('active');
     $('.js-title-'+module).parent('.configuration-rectangle').addClass('active');
     $('.js-title-'+module).parent('.configuration-rectangle').find('.module-informations').slideDown();
     $('.js-title-'+module+' .material-icons.up').show();
     $('.js-title-'+module+' .material-icons.down').hide();
 
-    headHeight = $('.page-head.with-tabs').height();
-    navHeight = $('#header_infos').height();
-    topOffset = headHeight + navHeight;
-    if (elem.hasClass('js-img-'+module)) {
-        $('html, body').animate({scrollTop: $('.js-title-'+module).offset().top-topOffset}, 1000);
-    } else {
-        if ($(window).innerWidth() > 991) {
-            $('html, body').animate({scrollTop: $('.js-img-'+module).offset().top-topOffset}, 1000);
-        }
-    }
+    // TODO: dirty fix
+    // wait for new position of active
+    setTimeout(() => {
+        $('html, body').animate({scrollTop: $('.configuration-rectangle.active').position().top}, 1000);
+    }, 300);
```

Comment:
No event can be caught instead? How can we be sure that 300ms is enough for environments?

Reply:
ye there is a `TODO: fix dirty fix` just above :D i've modified with a more graceful way